metrics/src/main/java/com/lightstep/tracer/metrics/CpuMetricGroup.java

```diff
@@ -0,0 +1,43 @@
+package com.lightstep.tracer.metrics;
+
+import java.io.IOException;
+
+import oshi.hardware.CentralProcessor.TickType;
+import oshi.hardware.HardwareAbstractionLayer;
+
+class CpuMetricGroup extends MetricGroup {
+  private static final CounterMetric<CpuMetricGroup,Long> cpuUsage = new CounterMetric<>("cpu.udage", Long.class);
+  private static final CounterMetric<CpuMetricGroup,Long> cpuTotal = new CounterMetric<>("cpu.total", Long.class);
+
+  CpuMetricGroup(final HardwareAbstractionLayer hal) {
+    super(hal, new CounterMetric<>("cpu.user", Long.class), null, new CounterMetric<>("cpu.sys", Long.class), null, null, null, null, null);
```

Comment:
Why so many `null` values here? Reserving spots for future/incoming metrics?

Reply:
The `Metric` instances in this array line up with the values [in this array](https://github.com/lightstep/lightstep-tracer-java-common/blob/0791b0e565232ea694a3d8c54dbad9b15c444dbc/metrics/src/main/java/com/lightstep/tracer/metrics/CpuMetricGroup.java#L18).